### PR TITLE
master-lps-103748

### PR DESCRIPTION
**Pull request by liuxingye**
https://issues.liferay.com/browse/LPS-103748

### CODE REVIEW

**Comment by liuxingye** [comment]
ci:test:sf

**Comment by liuxingye** [comment]
ci:test:sf

**Comment by austinchiang** [comment]
**Reviewing...** :hourglass_flowing_sand: :sunglasses: :headphones:

File: modules/apps/redirect/redirect-test/src/testFunctional/tests/Redirect.testcase [hunk 1]
```diff
@@ -357,6 +357,57 @@ definition {
...
+
+		Redirect.ignoreEntry(
+			ignoreURL = "${portalURL}/web/test-site-name/test-destination-page1",
+			Unignore = "true");
+
+		Redirect.viewURLsDefaultNotFoundCP();
+
+		Click(locator1 = "Dropdown#FILTER_AND_ORDER");
+
+		MenuItem.click(menuItem = "Active URLs");
```
Comment (austinchiang) [inline modules/apps/redirect/redirect-test/src/testFunctional/tests/Redirect.testcase:398]: We can probably turn this into a macro at some point. I'll push this test in for now

**Comment by austinchiang** [comment]
**Pull request submitted:** https://github.com/austinchiang/liferay-portal/pull/761<br></br>Please see changes [here.](https://github.com/austinchiang/liferay-portal/compare/07523e9c891a~1...07523e9c891a)

File: modules/apps/redirect/redirect-test/src/testFunctional/tests/Redirect.testcase (hunk 1 above)
Comment (liuxingye) [reply modules/apps/redirect/redirect-test/src/testFunctional/tests/Redirect.testcase:398]: I'll fix this in a test improvement ticket later. 

thanks @austinchiang

**Pull request closed without merging**